### DISCUSSION TITLE
FlorisBoard's new landing page and privacy policy

### DISCUSSION THREAD

**Discussion by patrickgold** [florisboard/florisboard#2021] [Announcements]
Starting today FlorisBoard has a simple landing page and new home for the privacy policy: https://florisboard.org/

It is a very simple website which will be reworked and expanded in the future (with the app store rebranding). The main reason for the creation isn't the landing page itself though, but the privacy policy needed to be rewritten and relocated to an "official, non-editable" location.

## What was wrong with the [old](https://gist.github.com/patrickgold/a18f1e47468d72f0868afc69d6faaf0b) privacy policy?

The old privacy policy violated the Google Play [User Data](https://support.google.com/googleplay/android-developer/answer/10144311?hl=en) policy according to Google. The main violation points are that the old location was not in an officially related source ( a Gist is not official), did not provide information about who is developing this keyboard and did not provide contact information for data-related inquiries.

## What has changed in the [new](https://florisboard.org/legal/privacy/) privacy policy?

For you as a user nothing has changed, FlorisBoard is still adhering to strict privacy protection and does not share your data at all. What has changed is the wording, I now disclose the inner workings of what personal data is accessed as well as locally stored (wouldn't be required from Google as the data remains always on the device but in my opinion it adds to the transparency of this keyboard) and I had to add the general legal texts.

That's it, nothing more. The new privacy policy will take effect with the next release v0.4.0-alpha03 for beta-testing users and with version v0.4.0 for stable users.

Feedback about this change is very welcome.

~ Patrick

**Comment by Efreak** [comment]
I'd like to note that the changelog says to check the detailed changelog for the explanation. The detailed changelog says to come here. Not really a big deal, but slightly annoying if you use apps--I opened the changelog in a github app (fasthub revived), but it doesn't support discussions (I don't think any github app does?) so I had to open the browser from there. Just something to be aware of in the future, maybe, as opening 3 new apps (fdroid, GitHub app, browser) will cause some low end devices to forget context in the original app.

It may also be worth noting in the privacy policy that without internet or storage permissions, the app is *incapable* of sending your data anywhere.

**Comment by pokegamer5547** [comment]
On google play store, the version of florisboard beta is still alpha 01. Did play store paused your updates cuz of privacy policy?

**Reply by patrickgold** [reply]
Yes, Google Play rejected alpha02 because of the old privacy policy issue and alpha03/04 is still stuck "In review" with the new privacy policy, so this is entirely on Google on how this proceeds further, I have done my part.
